sections/main-collection-product-grid.liquid

```diff
@@ -49,9 +49,7 @@
   
           <ul id="product-grid" data-id="{{ section.id }}" class="
             grid product-grid grid--{{ section.settings.columns_mobile }}-col-tablet-down
-            {% if section.settings.columns_desktop == 1 %} grid--full-width{% else %}
-              grid--{{ section.settings.columns_desktop }}-col-desktop
-            {% endif %}">
+            grid--{{ section.settings.columns_desktop }}-col-desktop
```

Comment:
Looks like a `>` got deleted here and is messing stuff up https://screenshot.click/25-07-16mb4-602jd.png